features/custom_parameter.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Custom Parameter
```

Comment:
Please use `parameter type` instead of `parameter` in this title, the feature description, and the scenario titles.